md/profileCreation.md

```diff
@@ -2,11 +2,23 @@
 
 ## Definition
 
-Basically, a **profile** is an XML file. It contains the profile name, a description, the mapping with the organization (which users / groups / roles / memberships have this profile) and the Bonita Portal menu entries (menus accessible to a user logged in the portal with this profile).  
-A profile must be imported onto the portal, in the **Profiles** tab, to enable the profile (you need to connect with the **Administrator** profile to do it).  
-Since Bonita 7.6.0, Bonita Studio offers this feature, for better packaging and deployment.
-For development purposes, the Studio can **deploy** profiles onto the portal for you (_only for Performance and Efficiency editions_).
-
+### Fonctional
+Profiles are used to grant some rights to some entities of the organization (users / groups / roles / memberships).
```

Comment:
I think we should simply the introduction by saying that profiles are used to grant some rights to users.
In a second sentence we can explain that user are associated with profiles either directly or by being members of groups with roles.

Reply:
ok